src/view/sidebar/sidebar-components.jsx

```diff
@@ -28,9 +29,13 @@ export const SharesComponent = ({ item }) => {
 		() =>
 			dispatch(getShareInfo()).then((res) => {
 				if (res.type.includes('fulfilled')) {
+					const calendars = uniqWith(
+						filter(res?.payload?.share ?? [], ['view', 'appointment']),
+						isEqual
+					);
 					const closeModal = createModal(
 						{
-							children: <SharesModal onClose={() => closeModal()} />
+							children: <SharesModal calendars={calendars} onClose={() => closeModal()} />
```

Comment:
it's better to use some other name like shares or sharedCalendar other then calendar only